wp-install test (#4)
granalacant/wp-azure-api-management: tests/install/README.md

# How to install it?
- Open your terminal and navigate to this directory.

> - Execute: docker-compose up -d

Reviewer: ```suggestion
- Execute: `docker-compose up -d`
```

> - Now, you can open a new tab on your explorer to: http://localhost:8080 A wordpress default page should be displayed.

Reviewer: Avoid `you` kind of phrases...
```suggestion
- Now, it is possible to open a new tab on any web browser to: http://localhost:8080 . A wordpress default page should be displayed.
```
